### ISSUE TITLE
As a user, I'd like the ability to add a custom background to jnet. 

### ISSUE THREAD

**Issue by kevintame** [mtgred/netrunner#4175]
This is an enhancements idea that I've had lots of people ask me about. People want the ability to have custom background on jnet. I don't think it is in your best interest to store background images but I think you can allow users to pull the images from another source. ie a linked image 

Check out the CSS from a chrome extension that I build. 

https://github.com/kevintame/jinteki-background-chrome-extension

Basically I think all you need to do is from the profile screen allow users to select "other/custom" from the background images options and then provide a field where the user can save a link to an external image.

Obviously not a priority but something if heard people ask about.

**Comment by francescopellegrini** [comment]
See #6413

**Comment by francescopellegrini** [comment]
Closed by #6413?
